.github/workflows/masterbot.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
+      - 'prerelease/**'
```

Comment:
Inspired by https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-to-specific-branches-occurs